RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -5710,6 +5710,12 @@ void auto_interruptCheck()
 	}
 }
 
+void auto_abort(string message)
```

Comment:
I dont think this is strictly necessary since autoscend.ash wraps the entry function with a try/finally: https://github.com/Loathing-Associates-Scripting-Society/autoscend/blob/f1dd671937e5aaf5eef495c8df9d8802345d758c/RELEASE/scripts/autoscend.ash#L15250

Reply:
This is mainly for when Ed aborts during combats. I'll try running without it & see if it's still an issue but it was just hard aborting before which was annoying sometimes.

Reply:
I could see that. Its not a terrible idea to have a wrapper anyway, gives a nice place to add helpful error handling stuff in one place. I had an idea for something that would be helpful here the other day but then went and forgot...maybe it will come back.

Reply:
Tested it & you're right, the try/finally catches it now. Removed.